### ISSUE TITLE
[Bug]: data problem with mo_catalog.mo_tables

### ISSUE THREAD

**Issue by sukki37** [matrixorigin/matrixone#7725]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): 70d7a4a
- Hardware parameters: 
- OS type:
- Others:
```

### Actual Behavior

All views in mo_catalog.mo_tables with empty rel_createsql field.
<img width="878" alt="image" src="https://user-images.githubusercontent.com/77312370/215328129-63dcc936-ea83-496d-8816-c389242d0973.png">

### Steps to Reproduce

```Markdown
create database test;
create table t1(a int);
create view v1 as select * from t1;
SELECT relname, rel_createsql,relkind FROM mo_catalog.mo_tables WHERE reldatabase='test' ;
```

**Comment by ouyuanning** [comment]
get it.

**Comment by aressu1985** [comment]
one cn node or cn-dn topo, this issue has been fixed:
mysql> SELECT relname, rel_createsql,relkind FROM mo_catalog.mo_tables WHERE reldatabase='test' ;
+-------------------------+----------------------------------------+---------+
| relname                 | rel_createsql                          | relkind |
+-------------------------+----------------------------------------+---------+
| %!%mo_increment_columns |                                        |         |
| t                       | CREATE TABLE t(f1 INTEGER)             | r       |
| v                       | CREATE VIEW v AS SELECT f1 FROM t      | v       |
| v0                      | ALTER VIEW v0 AS SELECT t00.a FROM t00 | v       |
| t1                      | create table t1(a int)                 | r       |
| v1                      | create view v1 as select * from t1     | v       |
+-------------------------+----------------------------------------+---------+
6 rows in set (0.01 sec)



but for 2 cn topo，it still exists：


mysql> SELECT relname, rel_createsql,relkind FROM mo_catalog.mo_tables WHERE reldatabase='test' ;
+-------------------------+---------------+---------+
| relname                 | rel_createsql | relkind |
+-------------------------+---------------+---------+
| %!%mo_increment_columns |               |         |
+-------------------------+---------------+---------+
1 row in set (0.01 sec)

**Comment by daviszhen** [comment]
It try it in the multi-CN-tae-DN config. It works well. Maybe there is no the database `test` in the 2CN node.

**Comment by aressu1985** [comment]
fixed:4410d861
